build/aznb.ps1

```diff
@@ -0,0 +1,33 @@
+# This script is useful for testing integration with Azure Notebooks.
```

Comment:
I think you're right it doesn't belong. Do you know where should it live, though? Life without this script was painful so I don't want to take it with me when I disappear into the ether

And that's a good point about Azure Notebooks setup. It's a little spicy; @xinyi-joffre had to send me some API key secrets needed to run it locally since I didn't have access. Not sure if other folks on our team would have the same problem